docker-entrypoint.sh

```diff
@@ -66,6 +66,10 @@ if [[ -n ${farmer_address} && -n ${farmer_port} ]]; then
   chia configure --set-farmer-peer "${farmer_address}:${farmer_port}"
 fi
 
+if [[ -n ${fullnode_address} && -n ${fullnode_port} ]]; then
+  chia configure --set-fullnode-peer "${fullnode_address}:${fullnode_port}"
```

Comment:
This isn't an available option for the `chia configure` command, at least as of `1.6.1`. Did you find the `--set-fullnode-peer` flag in a version of Chia that I may not be looking at?

Reply:
Ah, I have a habit of saying before reading. Reading now that you specified it requires changes from your fork. Since that's the case, this can be reviewed again when a PR is submitted to the chia-blockchain repo, and it's merged into the `latest` branch.